src/apps/bridge/sensor_drivers/softSensor.h

```diff
@@ -19,11 +19,12 @@ typedef struct SoftSensor : public AbstractSensor {
   AveragingSampler temp_deg_c;
   uint32_t reading_count;
 
-  // Extra sample padding to account for timing slop.
-  static constexpr uint32_t N_SAMPLES_PAD = 10;
+  // Extra sample padding to account for timing slop. Calculated as the sample frequency + 2 minutes bridge on period + some extra slop.
+  // 2 minutes is the minimum bridge on period and the soft by default is sampling at 2Hz. So 2*120 + 30 = 270.
+  static constexpr uint32_t N_SAMPLES_PAD = 270;
```

Comment:
Common aggregation settings include 15, 20, or even 30-minute sample durations, right? We need to gracefully handle someone setting that config too high. Would we go into a crash loop as the code is now?

Reply:
Sorry, I'm realizing this comment is on the pad, which I'm sure is fine. However, could you double check the allocation behavior if the sampling duration is set really high?

Reply:
This is mainly to handle the 2 minute BUS ON period after a bridge/spotter reset. The actual buffer sizes will be:
```
uint32_t AVERAGER_MAX_SAMPLES =(soft_agg_period_ms / _ctx.soft_reading_period_ms) + Soft_t::N_SAMPLES_PAD;
```
where 
```
soft_agg_period_ms = Sample duration (configurable)
_ctx.soft_reading_period_ms = period between soft readings (configurable)
```
^ the equation is from sensorController.cpp:191

Reply:
Now I don't know if this will cause us to crash from a lack of memory 😅. Thats the next ticket!

Reply:
Oh woops I didn't see your second comment! I can check that out before merging!

Reply:
Here are the heap stats with 1 SOFT and 1 Aanderaa with a 30 minute sample duration:
```
********* HEAP STATS *********
Available:            87352
Largest Free Block:   86832
Smallest Free Block:  24
Num free blocks:      9
Min ever free blocks: 86064
Total Allocs:         750
Total Frees:          475
Alloc/Free delta:     275
```
versus when I had it with a 3 minute sample duration for faster testing:
```
********* HEAP STATS *********
Available:            113816
Largest Free Block:   110792
Smallest Free Block:  24
Num free blocks:      12
Min ever free blocks: 106304
Total Allocs:         8012
Total Frees:          7733
Alloc/Free delta:     279
```
Resulting in a difference of 26,464 bytes.

Reply:
Awesomesauce. 🙌